### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by kodxana**
Made new readme and added prebuild binaries for Windows and MacOS

### CODE REVIEW

**Comment by btzr-io** [comment]
@kodxana we don't need the binaries in the repo, only source code, please learn how to make a release on github already post a link on slack for you, other than that looks good :+1:

File: README.md [hunk 1]
```diff
@@ -1,17 +1,43 @@
...
 
-[![David](https://img.shields.io/david/btzr-io/jelly-beats.svg?style=flat-square)](https://david-dm.org/btzr-io/jelly-beats)
-[![GitHub license](https://img.shields.io/github/license/btzr-io/jelly-beats.svg?style=flat-square)](https://github.com/btzr-io/electron-preact-app/blob/master/LICENSE)
+> What? A decentralized music streaming platform.
+How? We are developing an open source desktop application using the electron framework, react, web workers, unistore.js and the lbry-protocol.
+**Disclaimer:**
+We don’t intend to develop a real working product that can compete or replace other centralized platforms such as soundcloud, spotify, deezer (At least not right now).
+Our main goal right now it’s to develop a simple but functional application to showcase the current potential of the lbry-protocol for media distribution and discovery using simple web technologies.
```
Comment (btzr-io) [inline README.md]: Did you just copy paste this from the fund proposal ? :stuck_out_tongue:

**Comment by kodxana** [comment]
Removed binaries :)

File: README.md (hunk 1 above)
Comment (btzr-io) [reply README.md]: I don't think we need this ^^

Comment (kodxana) [reply README.md]: I was thinking its very cool :D

Comment (kodxana) [reply README.md]: @btzr-io Should be ok now :)

**Pull request merged**